pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -161,6 +161,18 @@ func createTargetConfigController_v311_00_to_latest(ctx context.Context, syncCtx
 		return true, err
 	}
 
+	if deprecatedPolicy {
+		upgradeableCondition := operatorv1.OperatorCondition{
+			Type:   "PolicyUpgradeable",
+			Status: operatorv1.ConditionFalse,
+			Reason: "PolicyFieldSpecified",
+			Message: fmt.Sprintf("scheduler.policy field is set which was deprecated and is to be removed in the next release"),
```

Comment:
Nit: `deprecated scheduler.policy field is set, and it is to be removed in the next release`